### ISSUE TITLE
Update copyright headers to appropriate ones.

### ISSUE THREAD

**Issue by JSAssassin** [w3c/vc-di-ecdsa-test-suite#27]
See - https://github.com/w3c/vc-di-ecdsa-test-suite/pull/24#discussion_r1440755371

**Comment by aljones15** [comment]
Additional Task:

"All Rights Reserved" should be removed.

**Comment by BigBlueHat** [comment]
Yes. Something to keep an eye on, of course, but the first pass is long done.
